### PR TITLE
Tickets/dm 12019

### PR DESCRIPTION
**Pull request by jgates108**

### CODE REVIEW

File: admin/tools/docker/deployment/kubernetes/kubectl/admin/yaml-builder.py [hunk 1]
```diff
@@ -222,16 +222,19 @@ def _add_emptydir_volume(volume_name):
...
         # Attach data-dir to containers
         #
-        # TODO add emptydir volume if config file is empty
+        data_volume_name = 'data-volume'
+        data_mount_path = '/qserv/data'
         if config.get('spec', 'host_data_dir'):
-            data_volume_name = 'data-volume'
-            data_mount_path = '/qserv/data'
             _add_volume(config.get('spec', 'host_data_dir'), data_volume_name)
+            # TODO use volume when master is splitted in 2 containers
```
Comment (jgates108) [inline admin/tools/docker/deployment/kubernetes/kubectl/admin/yaml-builder.py]: split

File: doc/source/install/k8s-sysadmin.rst [hunk 2]
```diff
@@ -16,8 +16,10 @@ Pre-requisites
    Cloud-Computing infrastructure. We provide Openstack support in pre-alpha
    version, please contact us for additional information.
 
+This tasks require sudo access and should be performed by system administrators.
```
Comment (jgates108) [inline doc/source/install/k8s-sysadmin.rst]: These

File: doc/source/install/k8s-sysadmin.rst [hunk 3]
```diff
@@ -39,58 +41,48 @@ Run multinode test
...
+.. literalinclude:: ../../../admin/tools/docker/deployment/kubernetes/env-infrastructure.example.in2p3.sh
    :language: bash
    :linenos:
 
-Run Openstack provisioning script and check `~/.lsst/qserv-cluster` directory to see examples.
-
-Then, install Kubernetes, Qserv and launch multinodes integration tests.
-
-.. code-block:: bash
+- an environment file named `env.sh` with all kubernetes configuration (qserv images, attached volumes, ...)
```
Comment (jgates108) [inline doc/source/install/k8s-sysadmin.rst]: Is there an env.sh.example?

File: doc/source/install/k8s-user.rst [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+For debugging purpose, it might be useful not to start Qserv services in pods:
+
+.. code-block:: bash
+
+   # on host machine
+   mkdir -p /qserv/custom/bin
+   touch /qserv/custom/bin/qserv-start.sh
+   chmod u+x /qserv/custom/bin/qserv-start.sh
+   # Qserv startup script will be here replaced by an empty file
```
Comment (jgates108) [inline doc/source/install/k8s-user.rst]: # Qserv startup script will be replaced by an empty file.
  
            
  Write

File: doc/source/install/k8s-sysadmin.rst (hunk 3 above)
Comment (fjammes) [reply doc/source/install/k8s-sysadmin.rst]: Yes, in admin/tools/docker/deployment/kubernetes

**Pull request merged**